src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -5,3 +5,62 @@ title: Working with Robots
+![servo close up](./images/ankle_leg/servo_close.jpg 'servo close up')
+There are three small indents on the silver disk, two on one side and one on the other. The single dot should be pointing towards the ground when the robot is standing and the other two dots towards the head.
+These three dots form an arrow, this arrow should be pointing downwards when the robot is standing.
+
+3. Manually move the disk on the servo so it is in the correct orientation 
+
+4. Place the plastic cover back on and replace all the screws. When replacing the screws, one hole must be left without a screw to allow for the servo to move. The hole left without a screw is the largest hole, which can be seen on the right of the following image:
+![screw hole](./images/angle_leg/screw_holes.jpg 'screw hole')
+
+## Replacing Shoulder servo
+
+1. Detatching the arm
+
+    - On the shoulder of the robot, there is a ring of screws front and back. Remove all the screws and the larger centre screw on both sides
+    - Remove the arm from the shoulder socket, it will still be attatched by a wire
+    - Take off the long part of plastic between the shoudler and neck, it is held by two screws. This piece of plastic can be seen in the following photo:
+    ![shoulder cover](./images/shoulder/shoulder.jpg 'shoulder cover')
+    - Now that the piece is off, there is a port for the chord that is still keeping the arm in place, detach this and the arm will be completely detatched
+2. Detatching the shoulder
+    - Looking in the shoulder socket there is another circle of screws, remove these and the shoulder will detatch
+    Note: Two plastic spacers should also fall out
+3. Removing the servo
+    - Remove and disconnect the boards and fans neccessary to access the shoulder servo from the inside, make sure to note any chords you detatch
+    - On the outside of the robot where the shoulder socket detatched, remove the four screws mounting the servo to the robots body
+    - There is a plastic holder inside the robot holding each shoulder servos in place, slide this towards the robots back to unclip the servos to allow it to fall freely out of its slot
+
+4. Mechanically testing the servo
+    - Remove the circular metal plate from the servo, get the plastic servo testor to test if the servo rotates freely without any clunks. If it doesn't rotate or is clunky, it needs replacing
+5. Installing a new servo
+    - Get the new servo set with the same serial number on the box
+    - Connect the metal circualr disk that came in the box on the new servo, ensuring you line up the indents on the metal disk and servo
+    - Place the servo into the body, put back the plastic holder and reconnect all the chords that were disconnected in the 'removing servo' step
+    - Rescrew the four screws that were taken out 
+    - The arms of the robot should be directly outwards when set to 0 degrees. This means you need to orientate the server before placing the shoulder back on. 
```

Comment:
Directly outwards? how do you mean?

Reply:
A picture of this would be very helpful given how many issues we had getting this right.

Reply:
I will get more images later this week and add them in